src/hb-face.cc

```diff
@@ -34,6 +34,31 @@
 #include "hb-ot-maxp-table.hh"
 
 
+
+/**
+ * hb_face_count: Get number of faces on the blob
+ * @blob:
+ *
+ *
+ *
+ * Return value: Number of faces on the blob
+ *
+ * Since: REPLACEME
+ **/
+unsigned int
+hb_face_count (hb_blob_t *blob)
+{
+  if (unlikely (!blob))
+    return 0;
+
+  hb_blob_t *sanitized = OT::Sanitizer<OT::OpenTypeFontFile> ().sanitize (blob);
+  const OT::OpenTypeFontFile& ot = *OT::Sanitizer<OT::OpenTypeFontFile>::lock_instance (sanitized);
+  if (unlikely (&ot == &OT::Null (OT::OpenTypeFontFile)))
```

Comment:
You should not need this, right? The whole idea of the Null objects is that you don't need to check for them constantly. Null object should already return 0 face count.

Reply:
Done. I guess it was used to make more sense on previous versions of the patch or not. Thanks